pysat/_constellation.py

```diff
@@ -136,6 +279,62 @@ def bounds(self, value=None):
 
         return
 
+    @property
+    def date(self):
+        """Date for loaded data."""
+        if len(self.index) > 0:
+            return utils.time.filter_datetime_input(self.index[0])
+        else:
+            return None
+
+    @property
+    def empty(self):
+        """Boolean flag reflecting lack of data, True if there is no Instrument
+        data in any Constellation Instrument.
+        """
+        return self._empty(all_inst=True)
+
+    @property
+    def index(self):
+        """Returns time index of loaded data."""
+        return self._index()
+
+    def today(self):
+        """Returns UTC date for today, see pysat.Instrument for details."""
+        return utils.time.today()
+
+    def tomorrow(self):
+        """Returns UTC date for tomorrow, see pysat.Instrument for details."""
+        return self.today() + dt.timedelta(days=1)
+
+    def yesterday(self):
+        """Returns UTC date for yesterday, see pysat.Instrument for details."""
+        return self.today() - dt.timedelta(days=1)
+
+    @property
+    def variables(self):
+        """Returns a list of uniquely named variables from all the loaded data.
+        """
+        # Determine which instrument variables share the same name
+        data_vars = dict()
+        for inst in self.instruments:
+            for dvar in inst.variables:
+                if dvar in data_vars.keys():
+                    data_vars[dvar] += 1
+                else:
+                    data_vars[dvar] = 1
+
+        # Distinguish shared names by Instrument platform, name, tag, inst_id
```

Comment:
Because it has whitespace and will be hard to use as a future data variable value.  I also hesitated to use `-` to separate.  Open to other suggestions, but keep in mind that in a file created after running the future `const_to_inst` these will be data variable names.